.travis.yml

```diff
@@ -46,8 +46,10 @@ matrix:
       env: NOX_SESSION=test-3.6
     - python: 3.7
       env: NOX_SESSION=test-3.7
-    - python: 3.8-dev
+    - python: 3.8
       env: NOX_SESSION=test-3.8
+    - python: 3.9-dev
```

Comment:
There's no 3.9-dev on Travis, but there's a "python-nightly" for both 16.04 and 18.04. Maybe that's 3.9-dev?